src/commands/dao_cmds/call.js

```diff
@@ -0,0 +1,113 @@
+const TaskList = require('listr')
+const Web3 = require('web3')
+const daoArg = require('./utils/daoArg')
+import initAragonJS from './utils/aragonjs-wrapper'
+const { listApps } = require('./utils/knownApps')
+const { rolesForApps } = require('./acl_cmds/utils/knownRoles')
+const { ensureWeb3 } = require('../../helpers/web3-fallback')
+import { addressesEqual } from './utils/web3-utils'
+
+const Table = require('cli-table')
+const colors = require('colors')
+
+const knownRoles = rolesForApps()
+const ANY_ENTITY = '0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF'
+const ANY_ENTITY_TEXT = 'Any entity'
+
+const path = require('path')
+let knownApps
+
+exports.command = 'call <dao> <proxy-address> <fn> [fn-args]'
```

Comment:
I think we should rename `call` to `exec`, so it doesn't conflict with the concept of a call in web3, which doesn't modify state.